playbooks/configure-apt-sources.yml

```diff
@@ -101,7 +101,18 @@
         dest: "/etc/openstack_deploy/group_vars/all/{{ item | basename }}"
       with_items:
         - files/apt.yml
-        - files/maas.yml
+      run_once: true
+      delegate_to: localhost
+      when:
+        - apt_artifact_found | bool
+        - apt_artifact_enabled | bool
+
+    - name: Sync artifact files (all)
+      copy:
+        src: "{{ item }}"
+        dest: "/etc/openstack_deploy/{{ item | basename }}"
+      with_items:
+        - files/user_rpcm_variables.yml
```

Comment:
How do we handle updating this file for the next release? The former releases clearly used the word 'defaults' and made the file read-only, giving a clear indication that upgrades of RPC-O would overwrite this file. This is putting the file into the 'you can edit it as you please' realm, so the question is how do we make changes when we upgrade?

Reply:
To ensure we own this file we could simply make it read-only and indicate via a comment, at the top of the file, that any addition will be removed upon upgrade. All that said, the old defaults are all deleted so there should be little risk, if any, from the current setup.

Reply:
done. see https://github.com/rcbops/rpc-openstack/pull/2805